CollAction/Controllers/AdminController.cs

```diff
@@ -250,7 +250,7 @@ public IActionResult Index()
                         "<br>" +
                         "The deadline of the project you have started on www.collaction.org has passed. We're very happy to see that the target you have set has been reached! Congratulations! Now it's time to act collectively!<br>" +
                         "<br>" +
-                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement (FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement ( FB: @<a href='https://www.facebook.com/collaction.org/'>collaction.org</a>, Twitter: @collaction_org)!<br>" +
```

Comment:
Also an extra space

Reply:
Don't we want the "@" inside the link?

Reply:
@ljwboschker this was requested, CA-318 :  
" 1. "word(FB: @collaction.org," --> add a space between "word" and the "(" "
@logophobia  this is now fixed